web/server/models.py

```diff
@@ -39,6 +39,39 @@ class ApplyType(str, enum.Enum):
     backfill = "backfill"
 
 
+class ConsoleEvent(str, enum.Enum):
+    """An enumeration of console events."""
+
+    plan_run = "plan-run"
+    plan_apply = "plan-apply"
+    tests = "tests"
+
+
+class Status(str, enum.Enum):
+    """An enumeration of statuses."""
+
+    init = "init"
+    success = "success"
+    fail = "fail"
+
+
+class PlanRunStage(str, enum.Enum):
+    """An enumeration of plan apply stages."""
+
+    validation = "validation"
+    changes = "changes"
+    backfills = "backfills"
+
+
+class PlanApplyStage(str, enum.Enum):
+    """An enumeration of plan apply stages."""
+
+    creation = "creation"
+    restate = "restate"
+    backfill = "backfill"
+    promote = "promote"
```

Comment:
Nit: I think plans are applied, but plans are not run/ran. Run is for running the DAG through the scheduler.